appveyor.yml

```diff
@@ -1,37 +1,30 @@
 ---
-image: Visual Studio 2017
+image: Visual Studio 2019
```

Comment:
Is it OK to bump this? 
(no idea if it is a problem, to be clear)

Reply:
It seems that 2017 and 2019 are both using C++ 14.x so they are all fine to use with Py35-39

Reply:
See [this up-to-date resource for CPython](https://wiki.python.org/moin/WindowsCompilers), which confirms this is a suitable change.